### PR TITLE
Add distribution tests, sync dependencies

### LINKED ISSUE #11: Add distribution tests

### LINKED ISSUE #12: Add dependencies from Plone package
As it should be possible to install `plone.classicui` instead of `Plone`, we have to make sure all packages listed there are also listed here.

### PR DESCRIPTION
**Pull request by ericof**
Fixes #11 
Fixes #12

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+	@echo "$(GREEN)==> Setup Build$(RESET)"
+	$(BIN_FOLDER)/mxdev -c mx.ini
+	$(BIN_FOLDER)/pip install -r requirements-mxdev.txt
+
+.PHONY: install
+install: install-plone-6 ## Install Plone 6
+
+.PHONY: start
+start: ## Start a Plone instance on localhost:8080
+	DEVELOP_DISTRIBUTIONS=$(DISTRIBUTIONS) PYTHONWARNINGS=ignore ./$(BIN_FOLDER)/runwsgi instance/etc/zope.ini
```
Comment (mauritsvanrees) [inline Makefile:102]: I do `make install` first, but then I cannot start:

```
$ make start
bash: line 1: .//Users/maurits/community/plone-coredev/6.1/src/plone.classicui/.venv/bin/runwsgi: No such file or directory
```

**Review by mauritsvanrees — COMMENTED** [review]
Seems okay, thanks.  Some issues with the Makefile.  And I asked my main question in the issue.

File: Makefile (hunk 1 above)
Comment (ericof) [reply Makefile:102]: Strange. This was working, now it is not

Comment (ericof) [reply Makefile]: Dumb mistake, should be solved now

**Comment by ericof** [comment]
@mauritsvanrees Any idea why tests are failing now?

**Comment by mauritsvanrees** [comment]
Probably explained by last remarks in release-team channel. Need new CMFPlone release.

**Comment by mauritsvanrees** [comment]
Or temporarily add CMFPlone in mxdev checkouts.

**Comment by mauritsvanrees** [comment]
I have made an internal release Products.CMFPlone 6.1.0a4.dev0 on https://dist.plone.org/release/6.1-dev/

But when I rerun the jobs, they [fail](https://github.com/plone/plone.classicui/actions/runs/9787320391/job/27157192049#step:8:79) with a weird error:

```
The conflict is caused by:
    pytest-plone 0.5.0 depends on Products.CMFPlone
    The user requested (constraint) products-cmfplone==6.1.0a4.dev0
```

Maybe this is a weird way for pip to say that it cannot find this distribution.  Could be that this needs to be added to a find-links somewhere.

**Comment by mauritsvanrees** [comment]
I have added find-links directly in the constraints file in dist.plone.org.  That seems to help for installation.

There are still errors though.

**Comment by mauritsvanrees** [comment]
I have removed most dependencies: multilingual, discussion, caching, iterate, CMFPlacefulWorkflow.  They are not needed, and @jensens and I are against adding them, see https://github.com/plone/plone.classicui/issues/12#issuecomment-2192732724

I did some back and forth importing, exporting and importing again, and see what changes this gave.  This led to some errors and then some fixes in exportimport, see https://github.com/plone/plone.exportimport/pull/35
Currently when I import this distribution and export it again, only these diffs appear:

* The `modified` dates are different.  Actually only two: for the news and events folders.  The date is likely changed because a Collection is added. :-/
* `__metadata__.json` gets an extra key `local_permissions`, with empty dicts for all content, except the Plone site root, which has seven local permissions.  This is probably best not added.

**Review by davisagli — DISMISSED** [review]

**Review by davisagli — APPROVED** [review]

**Review by jensens — APPROVED** [review]

**Pull request merged**